dev/Dockerfile_py3

```diff
@@ -0,0 +1,144 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Authors:
+# - Thomas Beermann <thomas.beermann@cern.ch>, 2017-2020
+# - Vincent Garonne <vgaronne@gmail.com>, 2017-2019
+# - Cedric Serfon <cedric.serfon@cern.ch>, 2017
+# - Frank Berghaus <frank.berghaus@cern.ch>, 2017-2018
+# - Mario Lassnig <mario.lassnig@cern.ch>, 2019
+
+FROM centos:7
+
+RUN yum -y update
+RUN yum -y install epel-release && \
+    yum -y install https://repo.ius.io/ius-release-el7.rpm && \
+    yum clean all && \
+    rm -rf /var/cache/yum
+RUN yum -y install \
+      gcc \
+      httpd \
+      mod_ssl \
+      mod_auth_kerb \
+      python36u-pip \
+      python36-devel \
+      python36-mod_wsgi \
+      python36-m2crypto \
+      gmp-devel \
+      krb5-devel \
+      git \
+      openssl-devel \
+      gridsite \
+      which \
+      libaio \
+      memcached \
+      nmap-ncat \
+      gfal2-all \
+      gfal2-util \
+      gfal2-python3 \
+      fts-rest-cli \
+      xrootd-client \
+      multitail \
+      vim && \
+      yum clean all && \
+      rm -rf /var/cache/yum
+
+RUN git clone https://github.com/rucio/rucio.git /tmp/rucio
+
+ENV RUCIOHOME=/opt/rucio
+RUN mkdir -p $RUCIOHOME
+RUN mkdir -p $RUCIOHOME/etc/multi_vo/tst/etc
+RUN mkdir -p $RUCIOHOME/etc/multi_vo/ts2/etc
+WORKDIR $RUCIOHOME
+RUN mkdir -p \
+      bin \
+      etc \
+      lib/rucio \
+      tools
+
+RUN python3 -m pip install --upgrade pip
+RUN python3 -m pip install --upgrade setuptools
+RUN rm -rf /usr/lib/python3.6/site-packages/ipaddress*
+RUN python3 -m pip install -r /tmp/rucio/etc/pip-requires-client
+RUN python3 -m pip install -r /tmp/rucio/etc/pip-requires
+RUN python3 -m pip install -r /tmp/rucio/etc/pip-requires-test
+RUN python3 -m pip install psycopg2-binary
+RUN ln -s $RUCIOHOME/lib/rucio /usr/local/lib/python3.6/site-packages/rucio
+
+COPY .pep8 .pep8
+COPY .flake8 .flake8
+COPY pylintrc etc/pylintrc
+RUN cp -r /tmp/rucio/etc .
+RUN cp -r /tmp/rucio/tools .
+
+RUN mkdir /var/log/rucio /var/log/rucio/trace
+RUN chmod -R 777 /var/log/rucio
+
+COPY rucio.cfg $RUCIOHOME/etc/rucio.cfg
+COPY rucio_multi_vo_tst.cfg $RUCIOHOME/etc/multi_vo/tst/etc/rucio.cfg
+COPY rucio_multi_vo_ts2.cfg $RUCIOHOME/etc/multi_vo/ts2/etc/rucio.cfg
+COPY alembic.ini $RUCIOHOME/etc/alembic.ini
+COPY aliases-py27.conf $RUCIOHOME/etc/web/aliases-py27.conf
+COPY ui-aliases-py27.conf $RUCIOHOME/etc/web/ui-aliases-py27.conf
+COPY google-cloud-storage-test.json $RUCIOHOME/etc/google-cloud-storage-test.json
+COPY certs/rucio_ca.pem /etc/grid-security/certificates/5fca1cb1.0
+
+COPY 00-mpm.conf /etc/httpd/conf.modules.d/00-mpm.conf
+COPY httpd.conf /etc/httpd/conf/httpd.conf
+COPY rucio.conf /etc/httpd/conf.d/rucio.conf
+COPY certs/ruciouser.pem $RUCIOHOME/etc/usercert.pem
+COPY certs/ruciouser.key.pem $RUCIOHOME/etc/userkey.pem
+COPY certs/ruciouser.certkey.pem $RUCIOHOME/etc/usercertkey.pem
```

Comment:
can be concatenated (if the files are named properly...) like this:
`COPY certs/usercert.pem certs/userkey.pem certs/usercertkey.pem $RUCIOHOME/etc/`